imports/plugins/included/actionable-analytics/client/templates/analyticsHighestSelling.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-unused-vars  */
+
+import { Template } from "meteor/templating";
+import Chart from "chart.js";
+import _ from "lodash";
+
+let productData = {};
+let productDetails = [];
+let productCounts = [];
+let productIds = [];
+let productNames = [];
+const backgroundColor = ["#FF6384", "#36A2EB", "#FFCE56", "#FE7EA6", "#2ACEEB"];
+
+const storeProductData = (productId) => {
+  if (productData[productId]) {
+    productData[productId] = productData[productId] + 1;
+  } else {
+    productData[productId] = 1;
+  }
+};
+
+const getProducts = (productId) => {
+  productId.forEach((id) => {
+    Meteor.call("analytics/getproduct", id, (error, result) => {
+      productNames.push(result[0].title);
+    });
+  });
+};
+
+const displayProducts = () => {
+  const ctx = document.getElementById("sellingProductChart");
+  const myChart = new Chart(ctx, {
+    type: "doughnut",
```

Comment:
It is not the product types but the type of the Pie Chart we chose to display the products. Meanwhile, we got some feedbacks from the product owner and product manager, so we are currently working to fix them.